pkg/controller/seed-controller-manager/encryption/controller.go

```diff
@@ -0,0 +1,290 @@
+/*
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package encryption
```

Comment:
Generally I think we name our controller packages `<thing>controller`. Not just because they all provide a `Add()` function and `encryption.Add()` doesn't make a lot of sense ^^

Reply:
We have like a 50:50 split for that, some controllers follow that scheme, some don't. But I'm happy to change that.